pkg/client/fake.go

```diff
@@ -63,9 +63,12 @@ func FakeInit() {
 	initEvent(kubeClient)
 }
 
-// SetDirectCSIClient sets fake direct-csi client.
-func SetDirectCSIClient(fakeClient *directcsifake.FakeDirectV1beta3) {
-	directCSIClient = fakeClient
+func SetUnversionedDirectCSIDriveClientset(driveInterface directcsiclientset.DirectCSIDriveInterface) {
+	directcsiDriveClientset = driveInterface
+}
+
+func SetUnversionedDirectCSIVolumeClientset(volumeInterface directcsiclientset.DirectCSIVolumeInterface) {
```

Comment:
can we remove this util function if it is not used anywhere else

Reply:
present to make the code symmetric